cpp/ql/src/Diagnostics/FailedExtractions.qll

```diff
@@ -0,0 +1,121 @@
+import cpp
+
+/**
+ * The class of errors upon we mark a file as non-successfully extracted.
```

Comment:
Please rephrase class-level QLDoc in this file to follow the style guide: https://github.com/github/codeql/blob/main/docs/qldoc-style-guide.md#qldoc-for-classes